solc/CommandLineInterface.cpp

```diff
@@ -941,6 +942,7 @@ bool CommandLineInterface::assemble(
 	yul::AssemblyStack::Language _language,
 	yul::AssemblyStack::Machine _targetMachine,
 	bool _optimize,
+	unsigned int _expectedExecutionsPerDeployment,
```

Comment:
Can you make this a `std::optional<unsigned>`, so that we can distinguish between the option being present and not?

Reply:
Actually I think this is an error in the original struct - it should be a `std::optional<unsigned>` already there. Also it is weird that it defaults to zero there.

Reply:
Huh, that's probably why the test fails. Hopefully making it an `std::optional<unsigned>` will fix that.